main.py

```diff
@@ -4,6 +4,11 @@
 import operator
 import letter_frequencies
 
+player = 0
+topScoreP1 = 0
+topScoreP2 = 0
+topWordP1 = "null"
+topWordP2 = "null"
```

Comment:
Instead of ```Null``` to assign an empty variable, you can simply use a pair of empty quotes like ```""``` or ```''```.

Reply:
variable names dont follow pep8

Reply:
also no globals please